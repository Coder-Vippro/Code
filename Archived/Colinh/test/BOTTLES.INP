6
6 10 10 13
10 10