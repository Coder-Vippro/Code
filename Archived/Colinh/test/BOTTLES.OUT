33 6
4
2
